`NettyMessagingServiceCompressionTest` tests fail to run locally

**Summary**

The tests fail locally with the following error if run from intellij
```
java.util.concurrent.CompletionException: io.atomix.cluster.messaging.MessagingException$ConnectionClosed: Channel [id: 0x289446b0] for address Mustafas-MacBook-Pro.local:1026 was closed unexpectedly before the request was handled

Caused by: io.atomix.cluster.messaging.MessagingException$ConnectionClosed: Channel [id: 0x289446b0] for address Mustafas-MacBook-Pro.local:1026 was closed unexpectedly before the request was handled
	at io.atomix.cluster.messaging.impl.NettyMessagingService.lambda$bootstrapClient$40(NettyMessagingService.java:772)
```

and if run from mvn, the build gets stuck forever until manually canceled

--- a/zeebe/atomix/cluster/src/test/java/io/atomix/cluster/messaging/impl/NettyMessagingServiceCompressionTest.java
+++ b/zeebe/atomix/cluster/src/test/java/io/atomix/cluster/messaging/impl/NettyMessagingServiceCompressionTest.java
@@ -33,7 +33,8 @@ class NettyMessagingServiceCompressionTest {
   @EnumSource(CompressionAlgorithm.class)
   void shouldSendAndReceiveMessagesWhenCompressionEnabled(final CompressionAlgorithm algorithm) {
     // given
-    final var senderAddress = Address.from(SocketUtil.getNextAddress().getPort());
+    var nextAddress = SocketUtil.getNextAddress();
+    final var senderAddress = Address.from(nextAddress.getHostName(), nextAddress.getPort());
     final var config =
         new MessagingConfig()
             .setShutdownQuietPeriod(Duration.ofMillis(50))
@@ -43,7 +44,8 @@ class NettyMessagingServiceCompressionTest {
         (ManagedMessagingService)
             new NettyMessagingService("test", senderAddress, config).start().join();
 
-    final var receiverAddress = Address.from(SocketUtil.getNextAddress().getPort());
+    nextAddress = SocketUtil.getNextAddress();
+    final var receiverAddress = Address.from(nextAddress.getHostName(), nextAddress.getPort());
     final var receiverNetty =
         (ManagedMessagingService)
             new NettyMessagingService("test", receiverAddress, config).start().join();

